packages/typespec-ts/test/integration/generated/mediaTypes/tspconfig.yaml

```diff
@@ -0,0 +1,13 @@
+emit:
+  - "@azure-tools/typespec-ts"
+options:
+  "@azure-tools/typespec-ts":
+    "emitter-output-dir": "{project-root}"
+    generateMetadata: false
+    generateTest: false
+    addCredentials: false
+    azureSdkForJs: false
+    isCadlTest: true
+    title: MediaTypesClient
+    packageDetails:
+      name: "@msinternal/media-types"
```

Comment:
Can you add create client test for these three cases ? just to make sure they can build successfully